src/test/kotlin/game/testClass/Rookie.kt

```diff
@@ -0,0 +1,24 @@
+package game.testClass
+
+import game.characters.Warrior
+import game.interfaces.BaseWarrior
+import game.settings.Params
+
+class Rookie : BaseWarrior {
+    private val attack: Int = 1
+    override fun hit(opponent: BaseWarrior) {
+        TODO("Not yet implemented")
+    }
+
+    override fun receiveDamage(damage: Int) {
+        TODO("Not yet implemented")
```

Comment:
is it still in progress?

Reply:
I will remove this class from this pull request, it is in use in the next one.